src/Tribe/Hooks.php

```diff
@@ -86,20 +88,55 @@ protected function add_actions() {
 	/**
 	 * Adds the filters required by the plugin.
 	 *
-	 * @since __TRIBE_VERSION__
+	 * @since 1.0.0
 	 */
 	protected function add_filters() {
+		add_filter( 'bulk_actions-edit-tribe_events', [ $this, 'modify_bulk_actions_label' ] );
+
+		add_action( 'manage_posts_extra_tablenav', [ $this, 'tec_ea_empty_ignored_button' ] );
 	}
 
 	/**
 	 * Load text domain for localization of the plugin.
 	 *
-	 * @since __TRIBE_VERSION__
+	 * @since 1.0.0
 	 */
 	public function load_text_domains() {
 		$mo_path = tribe( Plugin::class )->plugin_dir . 'languages/';
 
 		// This will load `wp-content/languages/plugins` files first.
-		\Tribe__Main::instance()->load_text_domain( 'tribe-ext-extension-template', $mo_path );
+		\Tribe__Main::instance()->load_text_domain( 'tribe-ext-ea-additional-options', $mo_path );
+	}
+
+	/**
+	 * Modify the 'Move to trash' bulk action label.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param array $bulk_actions Array of available bulk actions.
+	 *
+	 * @return array
+	 */
+	public function modify_bulk_actions_label( $bulk_actions ) {
+		if ( isset( $bulk_actions['trash'] ) ) {
+			$bulk_actions['trash'] = __( 'Move to Trash/Ignore', 'tribe-ext-ea-additional-options' );
+		}
+		return $bulk_actions;
+	}
+
+	/**
+	 * Create a "Permanently Delete All Ignored Events" button on the Ignored Events page.
+	 *
+	 * @since 1.5.0
+	 *
+	 * @return void
+	 */
+	public function tec_ea_empty_ignored_button(): void {
+		if (
+			tec_get_request_var( 'post_type') === 'tribe_events'
+			&& tec_get_request_var( 'post_status' ) === 'tribe-ignored'
+		) {
```

Comment:
Use yoda conditions...
```suggestion
		if (
			'tribe_events' === tec_get_request_var( 'post_type')
			&& 'tribe-ignored' === tec_get_request_var( 'post_status' )
		) {
```